<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
            QTableView {
                background-color: #ffffff; /* White background color */
                border: 1px solid #000000;
            }

            QHeaderView::section {
                background-color: #f2f2f2; /* Light gray background color */
                color: #000000; /* Black text color */
                border: none;
                padding: 8px;
                font-weight: bold;
            }

            QTableWidgetItem {
                padding: 4px;
            }
     
            QPushButton {
                background-color: #FFFFFF; /* White background color */
                color: #000000; /* Black text color */
                border: none;
                padding: 8px;
                font-size: 12px;
            }

            QPushButton:hover {
                background-color: #CCCCCC; /* Light gray background color on hover */
            }

            QPushButton:pressed {
                background-color: #999999; /* Dark gray background color when pressed */
            }
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
